modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -40,6 +40,14 @@ Tested Cloud Platforms for {product-title} 4.x: Amazon Web Services, Microsoft A
 | 25,000
 | 25,000
 
+| Number of routes and backends per Ingress Controller
+| 2,000 per router
+| 2,000 per router
+
+| Number of secrets and configmaps
+| 80000 secrets 90000 configmaps
+| 80000 secrets 90000 configmaps
```

Comment:
Consider adding a , in these metrics to be consistent with the rest of the table